content/docs/run-api-server/installing.mdx

```diff
@@ -5,25 +5,115 @@ order: 20
+  - [build Horizon yourself](#building) from scratch. 
 
-## Building
+The first method is recommended: not only do you ensure OS compatibility and dependency management, you'll also install some convenient wrappers that make running Horizon and Stellar-Core much simpler.
 
+### Package Manager
+SDF publishes new releases to its custom Ubuntu repositories. Follow [this guide](https://github.com/stellar/packages/blob/master/docs/adding-the-sdf-stable-repository-to-your-system.md#adding-the-sdf-stable-repository-to-your-system) to add the stable SDF repository to your system. Then, 
+
+<CodeExample>
+
+```bash
+sudo apt update && sudo apt install stellar-captive-core stellar-horizon
+```
+
+</CodeExample>
+
+[This page](https://github.com/stellar/packages/blob/master/docs/installing-individual-packages.md#installing-individual-packages) outlines the various commands that these packages make available.
+
+### Building
 Should you decide not to use one of our prebuilt releases, you may instead build Horizon from source. To do so, you need to install some developer tools:
 
-- A unix-like operating system with the common core commands (cp, tar, mkdir, bash, etc.)
+- A Unix-like operating system with the common core commands (cp, tar, mkdir, bash, etc.)
 - A compatible distribution of Go (Go 1.13 or later)
 - [git](https://git-scm.com/)
-- [mercurial](https://www.mercurial-scm.org/)
 
-1. See the details in [README.md](https://github.com/stellar/go/blob/master/README.md#dependencies) for installing dependencies.
-2. Compile the Horizon binary: `go install github.com/stellar/go/services/horizon`. You should see the `horizon` binary in `$GOPATH/bin`.
-3. Add Go binaries to your PATH in your `bashrc` or equivalent, for easy access: `export PATH=${GOPATH//://bin:}/bin:$PATH`
+**Note**: Though Horizon should run on Windows, *building* directly on Windows is not supported.
+
+First, refer to the details of [README.md](https://github.com/stellar/go/blob/master/README.md#dependencies) for installing dependencies. Then, you can build the Horizon binary pretty easily:
+
+<CodeExample>
+
+```bash
+git clone https://github.com/stellar/go monorepo && cd monorepo
+go install -v ./services/horizon
+sudo cp $(go env GOPATH)/bin/horizon /usr/bin/stellar-horizon
+```
+
+</CodeExample>
+
+_(You should refer to the list of [Horizon releases](https://github.com/stellar/go/releases) and `git checkout` accordingly if you're looking for a stable release rather than the bleeding edge `master` branch.)_
+
+At this point, you can either add Go binaries to your PATH in your `bashrc` or equivalent:
+
+<CodeExample>
+
+```bash
+export PATH=${GOPATH//://bin:}/bin:$PATH
+```
+
+</CodeExample>
+
+or just copy the binary from the `GOPATH` to the system PATH:
+
+<CodeExample>
+
+```bash
+cp $(go env GOPATH)/bin/horizon /usr/bin/stellar-horizon
+```
+
+</CodeExample>
+
+
+## Installing Remote Captive Core
```

Comment:
Can we add a bit about what a Remote Captive Core is and why someone might want to run one?  I think it will help people understand whether to pay attention to this section or skip it.

Reply:
I realized that this was covered on the "Configuring" page, which comes _after_ "Installing," so I've moved & updated this accordingly in 8d0b2bd2578c86413e2c5bd63f37b532d5fbf8d4.